lib/koine/repository/persistence/adapter.rb

```diff
@@ -2,31 +2,31 @@ module Koine
   module Repository
     module Persistence
       class Adapter
-        def exists?(criterias)
+        def exists?(*args)
           raise "Method not implemented: 'exists?'"
         end
 
-        def find_one_by(conditions)
+        def find_one_by(*args)
           raise "Method not implemented: 'find_one_by'"
         end
 
-        def update_where(conditions)
+        def update_where(*args)
           raise "Method not implemented: 'update_where'"
         end
 
-        def find_all_by(conditions)
+        def find_all_by(*args)
           raise "Method not implemented: 'find_all_by'"
         end
 
-        def find(id)
+        def find(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `find(*)` if you want the method to accept any arguments but don't care about them.